src/cyclic_queue.hpp

```diff
@@ -93,56 +87,43 @@ class CyclicQueue {
         return head[other];
     }
     /**
-     * @brief
-     * Enqueue
+     * @brief Enqueue
      *
      * @description
      * Enqueues the given value into the queue.
      * @param[in] T &value The value we wish to enqueue
```

Comment:
You still need to add a newline between the description and the parameters and/or return values everywhere.